iron-menu-behavior.html

```diff
@@ -23,7 +23,8 @@
   Polymer.IronMenuBehaviorImpl = {
 
     properties: {
-
+      _searchText: '',
```

Comment:
^ typed this before I saw your response, will dynamically initialize

i stand by my warnings ;)